infra/aws/terraform/management-account/accounts.tf

```diff
@@ -27,7 +27,7 @@ module "security_incident_response" {
   source = "../modules/org-account"
 
   account_name = "k8s-infra-security-incident-response"
-  email = "k8s-infra-aws-admins+incident-response@kubernetes.io"
+  email = "k8s-infra-aws-admins+security-incident-response@kubernetes.io"
```

Comment:
Is this going to cause drift? Will we / you fix it manually?

Reply:
Yes. We need to wait 90 days to use `k8s-infra-aws-admins+incident-response@kubernetes.io` to create a new account. Since we are bootstrapping the org, create another account should be ok.